src/main/java/com/axlsreborn/ProjectPropertiesFile.java

```diff
@@ -0,0 +1,62 @@
+package com.axlsreborn;
+
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Properties;
+
+public class ProjectPropertiesFile {
+    public static final String KEY_STEAM_WEB_API_KEY = "steam.web.api_key";
+    public static final String ID_STEAM_APP          = "steam.app_id";
+    public static final String CSV_DELIMITER         = "csv.delimiter";
+    public static final String FILE_PATH_ACHIEVEMENTS = "file.path_achievements";
+
+    public static String[] readFile(String[] args) {
+        String[] projectProperties = new String[4];
+
+        if (args.length != 1) {
+            System.err.println("You must specify the path to the project properties file");
+            System.exit(1);
+        }
+
+        // Steam Web API Key and App ID File Input
+        String propertiesPath = args[0];
+        System.out.println("Main.main(): propertiesPath = [" + propertiesPath + "]");
+        try {
+            FileReader fileReader = new FileReader(propertiesPath);
+            Properties properties = new Properties();
+            properties.load(fileReader);
+
+            String apiKey = properties.getProperty(KEY_STEAM_WEB_API_KEY);
+            String appID = properties.getProperty(ID_STEAM_APP);
```

Comment:
```suggestion
            int appId = ...
```
You should parse the `appId` into an int instead of keeping it as a string.

You also want it to have a lowercase `d` to follow the camelCase convention. If an acronym is used in a camelCase variable name, you should lowercase all letters that aren't the first one. For example, you have a variable that holds an HTML result from an HTTP call.
`String HttpHtmlResult = "<h1>Yup</h1>";`